### DISCUSSION TITLE
shapeworks如何将对齐后的vtk格式下载下来

### DISCUSSION THREAD

**Discussion by zestyzy** [SCIInstitute/ShapeWorks#2278] [Q&A] [answered]
在shapeworks中使用align对齐了并且groomed后发现也对齐，但是输出为groomed.vtk并在后续的形状模型训练中发现他并没有实现对齐

**Comment by akenmorris** [comment, marked as answer]
Yes, the alignment is not applied to the meshes for a number of reasons.  See here:

* #2271
